config/locales/en/lookup.yml

```diff
@@ -65,16 +65,16 @@ en:
         wales:
           guidance:
             label: Find out what you can or cannot do (Welsh Government)
-            link: "https://gov.wales/local-lockdown"
+            link: "https://gov.wales/coronavirus-regulations-guidance"
         northern_ireland:
           guidance:
             label: Find out what you can or cannot do (Northern Ireland Government)
             link: "https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-and-localised-restrictions"
       no_information:
         heading: There is no information about the restrictions in this area
         guidance:
-          label: Full list of Local COVID Alert Levels by area
-          link: "/guidance/full-list-of-local-covid-alert-levels-by-area"
+          label: Full list of tiers by area
+          link: "https://www.gov.uk/guidance/full-list-of-local-restriction-tiers-by-area"
```

Comment:
This link "https://www.gov.uk/guidance/full-list-of-local-restriction-tiers-by-area" currently gives a 404, I assume this is because we don't have the full list ready yet?

Reply:
ah yes, it's not yet published... hmmmm let me change that